extension/neo4j/test/test_files/doc_example.test

```diff
@@ -17,3 +26,33 @@
 -STATEMENT match (u:User)-[f:Follows]->(u1:User) return max(f.since);
 ---- 1
 2022
+
+-CASE MigrateFromNeo4jDocSampleUsingKleeneStar
+-STATEMENT load extension "${KUZU_ROOT_DIRECTORY}/extension/neo4j/build/libneo4j.kuzu_extension"
+---- ok
+-STATEMENT CALL NEO4J_MIGRATE("localhost", "neo4j", "czy990424", ["*"], ["*"]);
```

Comment:
WOuld be nice to have the other single kleen star variations tested in this file too.